docs/vercel-ko.md

```
@@ -33,7 +40,9 @@
 ⚠️️ 참고: 환경 변수를 변경할 때마다 [프로젝트를 재배포](#如何重新部署)해야 변경 사항을 적용할 수 있습니다!
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)